rpc/lib/rpc_test.go

```diff
@@ -144,7 +144,7 @@ func setup() {
 	time.Sleep(time.Second * 2)
 }
 
-func echoViaHTTP(cl client.HTTPClient, val string) (string, error) {
```

Comment:
We should add a call to `SetCodec` probably for it to stay `client.HTTPClient`

Reply:
This was an accidental change. I reverted this and the others like it.